boxes/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "create-aztec-app",
   "packageManager": "yarn@4.1.0",
-  "version": "0.2.15",
+  "version": "0.3.16",
```

Comment:
Is this version string maintained manually?

Reply:
yes, I'm a lazy guy lol I think it's not worth setting up release-please as we expect this script to not change that much...